### PR TITLE
update german language

### PR DESCRIPTION
**Pull request by SkavenKF**
## Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [x] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: languages/de_de/Home.php [hunk 1]
```diff
@@ -1,12 +1,11 @@
 <?php
-/* +***********************************************************************************
- * The contents of this file are subject to the vtiger CRM Public License Version 1.0
- * ("License"); You may not use this file except in compliance with the License
- * The Original Code is:  vtiger CRM Open Source
- * The Initial Developer of the Original Code is vtiger.
- * Portions created by vtiger are Copyright (C) vtiger.
- * All Rights Reserved.
- * *********************************************************************************** */
+/**
```
Comment (mariuszkrzaczkowski) [inline languages/de_de/Home.php]: You can not delete an old license, we only add this license to new files.

File: languages/de_de/Settings/Roles.php [hunk 2]
```diff
@@ -1,12 +1,11 @@
 <?php
-/* +***********************************************************************************
- * The contents of this file are subject to the vtiger CRM Public License Version 1.0
- * ("License"); You may not use this file except in compliance with the License
- * The Original Code is:  vtiger CRM Open Source
- * The Initial Developer of the Original Code is vtiger.
- * Portions created by vtiger are Copyright (C) vtiger.
- * All Rights Reserved.
- * *********************************************************************************** */
+/**
```
Comment (mariuszkrzaczkowski) [inline languages/de_de/Settings/Roles.php]: You can not delete an old license, we only add this license to new files.

File: languages/de_de/Settings/SharingAccess.php [hunk 3]
```diff
@@ -1,12 +1,11 @@
 <?php
-/* +***********************************************************************************
- * The contents of this file are subject to the vtiger CRM Public License Version 1.0
- * ("License"); You may not use this file except in compliance with the License
- * The Original Code is:  vtiger CRM Open Source
- * The Initial Developer of the Original Code is vtiger.
- * Portions created by vtiger are Copyright (C) vtiger.
- * All Rights Reserved.
- * *********************************************************************************** */
+/**
```
Comment (mariuszkrzaczkowski) [inline languages/de_de/Settings/SharingAccess.php]: You can not delete an old license, we only add this license to new files.

Comment (SkavenKF) [reply languages/de_de/Settings/SharingAccess.php]: ok, sorry. should i I undo this?

File: languages/de_de/Settings/Roles.php (hunk 2 above)
Comment (SkavenKF) [reply languages/de_de/Settings/Roles.php]: ok, sorry. should i I undo this?

File: languages/de_de/Home.php (hunk 1 above)
Comment (SkavenKF) [reply languages/de_de/Home.php]: ok, sorry. should i I undo this?

File: languages/de_de/Settings/SharingAccess.php (hunk 3 above)
Comment (mariuszkrzaczkowski) [reply languages/de_de/Settings/SharingAccess.php]: yes

File: languages/de_de/Settings/Roles.php (hunk 2 above)
Comment (mariuszkrzaczkowski) [reply languages/de_de/Settings/Roles.php]: yes

**Pull request merged**